addons/sale_timesheet/models/project_task.py

```diff
@@ -86,11 +86,18 @@ def _get_last_sol_of_customer(self):
         if not self.partner_id.commercial_partner_id or not self.allow_billable:
             return False
         SaleOrderLine = self.env['sale.order.line']
+        commercial_partner = self.partner_id.commercial_partner_id
+        if hasattr(commercial_partner, '_origin') and commercial_partner._origin:
+            commercial_partner_id = commercial_partner._origin.id
+        else:
+            commercial_partner_id = commercial_partner.id
+        if not isinstance(commercial_partner_id, int):
+            return False
         domain = expression.AND([
             SaleOrderLine._domain_sale_line_service(),
             [
                 ('company_id', '=?', self.company_id.id),
-                ('order_partner_id', 'child_of', self.partner_id.commercial_partner_id.id),
+                ('order_partner_id', 'child_of', commercial_partner_id),
```

Comment:
Cannot we just do that:

```suggestion
                ('order_partner_id', 'child_of', self.partner_id.commercial_partner_id.ids),
```
? 🤔

Reply:
you're right, it worked